horizons/world/player.py

```diff
@@ -69,7 +69,8 @@ def initialize(self, inventory):
 
 	def __init(self, name, color, clientid, difficulty_level, max_tier_notification, settlerlevel=0):
 		assert isinstance(color, Color)
-		assert isinstance(name, basestring) and name
+		assert isinstance(name, basestring)
+		assert name
```

Comment:
A question about type hints: Isn't that basically the same as doing `isinstance()` all over?